templates/macros/cycle-select.html

```diff
@@ -1,6 +1,6 @@
-{% macro cycle_select(cycles, cycle, location='query') %}
+{% macro cycle_select(cycles, cycle, location='query', id='cycle-select') %}
 {% if cycles %}
-  <select class="cycle-select select--neutral js-cycle" aria-label="Two-Year Period" name="cycle" data-cycle-location="{{ location }}">
+  <select id="{{ id }}" class="cycle-select select--neutral js-cycle" aria-label="Two-Year Period" name="cycle" data-cycle-location="{{ location }}">
```

Comment:
Do we need the `id` here for accessibility reasons? It doesn't look like we're using it in the code.